src/tests/roc_audio/test_profiler.cpp

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <CppUTest/TestHarness.h>
+
+#include "roc_audio/profiler.h"
+#include "roc_core/heap_allocator.h"
+
+namespace roc {
+namespace audio {
+
+namespace {
+struct TestFrame {
+    TestFrame(size_t sz, core::nanoseconds_t tm)
+        : size(sz)
+        , time(tm) {
+    }
+
+    size_t size;
+    core::nanoseconds_t time;
+};
+
+const int sampleRate = 5000; // 50 samples / chunk
```

Comment:
Nitpick: camel case constants and class names starts with capital letter, i.e. `SampleRate`. I think I should document all these little conventions...